src/apiCalls.js

```diff
@@ -15,14 +37,24 @@ usersData = users.users
 ingredientsData = ingredients.ingredients
 
 showRecipesPage();
-user = selectRandomUser(usersData)
-displayRecipes(recipeData)
-return user
-})})
+user = selectRandomUser(usersData);
+displayRecipes(recipeData);
+return user;
+})});
+
+fetch(`https://api.frankfurter.app/currencies`)
+  .then(data => data.json())
+  .then((data) => {
+    const entries = Object.entries(data);
+    entries.forEach((entry) => {
+      currency[0].innerHTML += `<option value="${entry[0]}">${entry[0]}</option>`;
+    })
+  })
+  .catch(error => alert(`${error.message}`));
 
 export {
     recipeData,
     usersData,
     ingredientsData,
-    user
+    user,
 }
```

Comment:
Can we remove the comma on line 27?

Reply:
Additional exports have been added below this line, so this is now resolved :)